sgtm/artemis/README.md

```diff
@@ -0,0 +1,273 @@
+# Artemis
+
+Get documentes from [GCP Firestore](https://firebase.google.com/docs/firestore) 
```

Comment:
Nits: 
- typo "get documents".
- Use Google Cloud not GCP (as [the name was rebranded](https://cloud.google.com/blog/topics/developers-practitioners/introducing-new-homepage-google-cloud))

Reply:
Updated